.github/workflows/ci.yml

```diff
@@ -59,7 +62,7 @@ jobs:
           docker network create --attachable cloudbuild
       - name: Docker Snuba tests
         run: |
-          SNUBA_IMAGE=getsentry/snuba docker-compose -f docker-compose.gcb.yml run --rm snuba-test
+          SNUBA_IMAGE=getsentry/snuba SNUBA_MODE=${{ matrix.ci_settings }} docker-compose -f docker-compose.gcb.yml run --rm snuba-test
```

Comment:
Can we just refer to this as `snuba_settings` or `SNUBA_SETTINGS` everywhere? Instead of ci_settings, SNUBA_MODE and SNUBA_SETTINGS in various places.